backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt

```diff
@@ -16,10 +16,14 @@ import org.jetbrains.exposed.sql.leftJoin
 import org.jetbrains.exposed.sql.select
 import org.jetbrains.exposed.sql.selectAll
 import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+import org.kodein.di.LazyDI
+import org.kodein.di.instance
 import pw.forst.tools.katlib.toUuid
 import java.util.UUID
 
-class PatientService {
+class PatientService(di: LazyDI) {
+    val validationService by di.instance<ValidationService>()
```

Comment:
jsem spis priznivcem toho to posilat pres konstruktor rovnou tzn:
```kotlin
class PatientService(private val validationService)
```

a pak v DI kontejneru 
```kotlin
bind<PatientService> with singleton { PatientService(instance()) }
```

Reply:
Jo, ja to opsal odnekud z kodu :)

Reply:
tak to asi pravdepodobne odnekud z `route`, tam se to da dostat prave takhle, ve zbytku apky by to tam melo lezt pres kontruktor aby to o nejakem DI vubec nevedelo

pak se musi pridat prave inicializace instance do DI kontejneru v `DependencyInjection.kt`